mage/tutorials/create-a-new-module-python-local.mdx

```diff
@@ -0,0 +1,276 @@
+
+```
+docker cp . mage:/mage/
+```
+
+**b)** Then you need to position yourself inside container as root:
+
+```
+docker exec -u root -it mage /bin/bash
+```
+
+> Note: if you have done build locally, make sure to delete folder `cpp/build` because
+> you might be dealing with different `architectures` or problems with `CMakeCache.txt`.
+> To delete it, run:
+>
+> `rm -rf cpp/build`
+
+**c)** After that run build with option to copy executables from `mage/dist` to `/usr/lib/memgraph/query_modules`:
+
+```
+python3 setup build -p /usr/lib/memgraph/query_modules/
+```
+**d)** Everything is done now. Just run following command:
+
+```
+exit
+```
+** And that is it.**
+
+  </TabItem>
+
+
+  <TabItem value="docker">
+
+**1.** To create `dev` **MAGE** image, run the following command:
+
+```
+docker build --target dev -t memgraph-mage:dev .
+```
+**2.** Now run the image with following command:
+
+```
+docker run --rm -p 7687:7687 --name mage memgraph-mage:dev
+```
+
+**And that is it.**
+
+:::info
+Now if you make any additional changes, you can stop container and do the rebuild.
+Additionally if you don't want to rebuild everything again, you can have running container and copy stuff inside
+container, do build inside container, and copy executables to  /usr/lib/memgraph/query_modules/ folder,
+where Memgraph is looking for query modules. You can follow these steps for any additional changes:
+
+**a)** First you need to copy files to container with name `mage`
+
+```
+docker cp . mage:/mage/
+```
+
+**b)** Then you need to position yourself inside container as root:
+
+```
+docker exec -u root -it mage /bin/bash
+```
+
+> Note: if you have done build locally, make sure to delete folder `cpp/build` because
+> you might be dealing with different `architectures` or problems with `CMakeCache.txt`.
+> To delete it, run:
+>
+> `rm -rf cpp/build`
+
+**c)** After that, run build and copying `mage/dist` to `/usr/lib/memgraph/query_modules`:
+
+```
+python3 setup build -p /usr/lib/memgraph/query_modules/
+```
+**d)** Everything is done now. Just run following command:
+
+```
+exit
+```
+:::
+
+  </TabItem>
+
+  <TabItem value="source">
+**0.** Make sure your Memgraph instance is running.
+
+```
+sudo systemctl status memgraph.service
+```
+
+**1.** Now, we need to copy our developed query module `random_walk` to `/usr/lib/memgraph/query_modules`.
+
+```
+python3 setup build -p /usr/lib/memgraph/query_modules
+```
+
+And that is it.
+  </TabItem>
+</Tabs>
+
+
+## Querying
+
+> Note that query modules are loaded into Memgraph on startup so if your instance was already running you will need to
+> execute the following query inside one of [querying platforms](https://docs.memgraph.com/memgraph/connect-to-memgraph) to load them:
+```
+CALL mg.load_all();
+```
+
+Lastly, run a query and test your module:
+
+```
+MERGE (start:Node {id: 0})-[:RELATION]->(mid:Node {id: 1})-[:RELATION]->(end:Node {id: 2})
```

Comment:
Why we have mid and end if we are not using it :sweat_smile:

Reply:
Removed `mid` and `end`.